Add a kill switch for using tr1 unordered_map.  Gcc on some systems,
eg, module at05 on biou, are version >= 4.3 but don't have unordered
map.

diff -Narub symtabAPI.orig/src/dyninst/dynutil/h/dyntypes.h symtabAPI/src/dyninst/dynutil/h/dyntypes.h
--- symtabAPI.orig/src/dyninst/dynutil/h/dyntypes.h	2009-12-08 12:03:01.000000000 -0600
+++ symtabAPI/src/dyninst/dynutil/h/dyntypes.h	2011-02-26 13:00:12.000000000 -0600
@@ -55,8 +55,8 @@
   #include <functional>
   #define DECLTHROW(x) throw(x)
   //***************** GCC ***********************
-   #if (__GNUC__ > 4) || \
-      (__GNUC__ == 4 && __GNUC_MINOR__ >= 3)
+   #if ( (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ >= 3) ) \
+       && (! defined(DISABLE_TR1_MAP) )
       //**************** GCC >= 4.3.0 ***********
       #include <tr1/unordered_set>
       #include <tr1/unordered_map>
